docs/source/using/index.adoc

```diff
@@ -6,6 +6,7 @@ This section describes how to manage your Minishift VM and interact with your
 OpenShift cluster. It also helps troubleshooting common issues.
 
 - link:../using/managing-minishift{outfilesuffix}[Managing Minishift]
+- link:../using/proxies{outfilesuffix}[HTTP/HTTPS Proxies]
```

Comment:
I am not sure whether I am in favour to break all sections out. I think having the basic life cycles methods and things like HTTP proxy in one file makes sense. we start to loose any sort of grouping here and just add any odd topic to the highest level.